include/boost/geometry/strategies/convex_hull/cartesian.hpp

```diff
@@ -0,0 +1,49 @@
+// Boost.Geometry
+
+// Copyright (c) 2020, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_CARTESIAN_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_CARTESIAN_HPP
+
+
+#include <boost/geometry/strategies/convex_hull/services.hpp>
+#include <boost/geometry/strategies/detail.hpp>
+#include <boost/geometry/strategy/cartesian/side_robust.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategies { namespace convex_hull
+{
+
+template <typename CalculationType = void>
+class cartesian : strategies::detail::cartesian_base
+{
+public:
+    static auto side()
+    {
+        return strategy::side::side_robust<CalculationType>();
```

Comment:
So the default side strategy is robust strategy. Is that correct? Typically we're choosing the fastest as the default. Does it mean that the robust side strategy is also faster?

Reply:
> So the default side strategy is robust strategy. Is that correct? Typically we're choosing the fastest as the default. Does it mean that the robust side strategy is also faster?

Yes, I chose to put robust as default one. It seems that it is also faster, see this discussion https://github.com/boostorg/geometry/issues/699

> EDIT: Is it because robust convex_hull is needed in e.g. in triangulation? If that's the case then convex_hull umbrella strategy could define non-robust strategy and triangulation umbrella strategy could define robust one.

No, this is independent from triangulation.